flake.nix

```diff
@@ -0,0 +1,40 @@
+{
+  description = "A Nix flake for NumScript WASM compilation";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        inherit (pkgs.lib) optional optionals;
+        pkgs = import nixpkgs { inherit system; };
+
+        inputs = with pkgs; [
+          go
+          tinygo
+          git
+          act
+          antlr4
+          just
+          nodejs_20
+          tree
```

Comment:
como seriam usados `nodejs` e `tree` com numscript? e `just`? e `act`?

Reply:
@debora-be